images/linux/scripts/installers/bazel.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+################################################################################
+##  File:  bazel.sh
+##  Desc:  Installs bazel
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Install bazel
+apt-get install -y bazel
+
+# Run tests to determine that the software installed as expected
+echo "Testing to make sure that script performed as expected, and basic scenarios work"
+if ! command -v bazel; then
+    echo "Bazel was not installed"
+    exit 1
+fi
+
+# Document what was added to the image
+echo "Lastly, documenting what we added to the metadata file"
```

Comment:
nitpick: not sure we need echo for docs

Reply:
We should, at least to be consistent with other tools.
For example, AzCopy: https://github.com/actions/virtual-environments/blob/master/images/linux/scripts/installers/azcopy.sh

Reply:
Yeah it totally makes sense to be consistent, just wondering why we need echo for docs.